### ISSUE TITLE
Add section numbers to the sections in style.css for quick jumping to a section

### ISSUE THREAD

**Issue by BraadMartin** [MIGHTYminnow/trestle#38]
For example:

````
/* 1.2 - Box Sizing
--------------------------------------------- */
````

**Comment by MickeyKay** [comment]
Yeah, I hear ya'. This is one I've wondered about maintaining. The issue is that if you add a new section in development (e.g. Custom Plugin Styles), you may need to renumber all the sections.

**Comment by BraadMartin** [comment]
I see the point about adding new sections, but I think it's less important that we keep custom sections perfect than it is to just have the core sections be numbered, and if we find ourselves always adding the same custom section here or there, maybe that section belongs in core Trestle.

That said, it looks like underscores has switched away from numbering sections, so maybe we shouldn't make the change. I want to take a step back and reconsider the organization of the stylesheet as a whole  at some point, ideally with your perspective included, so I'll rename this issue to be more about that and we can use this to track ideas until we make a final decision.
